examples/coap-server.c

```diff
@@ -991,6 +994,8 @@ main(int argc, char **argv) {
         exit(1);
       }
       break;
+    case 'n':
+      require_peer_cert = 0;
```

Comment:
There is a break statement missing here.

Reply:
Oops - now fixed